From a819c0003c8a2918bacf93193903c9a117b63b86 Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Mon, 22 Oct 2012 18:17:37 +0800
Subject: [PATCH 356/505] ENGR00230647 camera: corrent the example command
 line for mxc_v4l2_overlay

all the command is miswriten as mxc_v4l2_capture
place mxc_v4l2_capture with mxc_v4l2_overlay
---
 test/mxc_v4l2_test/doc/FSL-UT-V4L2-008.txt | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/test/mxc_v4l2_test/doc/FSL-UT-V4L2-008.txt b/test/mxc_v4l2_test/doc/FSL-UT-V4L2-008.txt
index 1d37546..fb270a0 100644
--- a/test/mxc_v4l2_test/doc/FSL-UT-V4L2-008.txt
+++ b/test/mxc_v4l2_test/doc/FSL-UT-V4L2-008.txt
@@ -33,35 +33,35 @@ Type commands to direct preview the camera to the foreground/background, and set
 
 Test foreground display:
 
-Test 1. mxc_v4l2_capture.out -iw 640 -ih 480 -ow 640 -oh 480 -m 0 -fg
+Test 1. mxc_v4l2_overlay.out -iw 640 -ih 480 -ow 640 -oh 480 -m 0 -fg
 
-Test 2. mxc_v4l2_capture.out -iw 320 -ih 240 -ow 640 -oh 480 -m 1 -fg
+Test 2. mxc_v4l2_overlay.out -iw 320 -ih 240 -ow 640 -oh 480 -m 1 -fg
 
-Test 3. mxc_v4l2_capture.out -iw 720 -ih 480 -ow 640 -oh 480 -m 2 -fg
+Test 3. mxc_v4l2_overlay.out -iw 720 -ih 480 -ow 640 -oh 480 -m 2 -fg
 
-Test 4. mxc_v4l2_capture.out -iw 720 -ih 576 -ow 640 -oh 480 -m 3 -fg
+Test 4. mxc_v4l2_overlay.out -iw 720 -ih 576 -ow 640 -oh 480 -m 3 -fg
 
-Test 5. mxc_v4l2_capture.out -iw 1280 -ih 720 -ow 640 -oh 480 -m 4 -fg
+Test 5. mxc_v4l2_overlay.out -iw 1280 -ih 720 -ow 640 -oh 480 -m 4 -fg
 
-Test 6. mxc_v4l2_capture.out -iw 1920 -ih 1080 -ow 640 -oh 480 -m 5 -fg
+Test 6. mxc_v4l2_overlay.out -iw 1920 -ih 1080 -ow 640 -oh 480 -m 5 -fg
 
-Test 7. mxc_v4l2_capture.out -iw 2592 -ih 1944 -ow 640 -oh 480 -m 6 -fg
+Test 7. mxc_v4l2_overlay.out -iw 2592 -ih 1944 -ow 640 -oh 480 -m 6 -fg
 
 Test background display:
 
-Test 8. mxc_v4l2_capture.out -iw 640 -ih 480 -ow 640 -oh 480 -m 0 -bg
+Test 8. mxc_v4l2_overlay.out -iw 640 -ih 480 -ow 640 -oh 480 -m 0 -bg
 
-Test 9. mxc_v4l2_capture.out -iw 320 -ih 240 -ow 640 -oh 480 -m 1 -bg
+Test 9. mxc_v4l2_overlay.out -iw 320 -ih 240 -ow 640 -oh 480 -m 1 -bg
 
-Test 10. mxc_v4l2_capture.out -iw 720 -ih 480 -ow 640 -oh 480 -m 2 -bg
+Test 10. mxc_v4l2_overlay.out -iw 720 -ih 480 -ow 640 -oh 480 -m 2 -bg
 
-Test 11. mxc_v4l2_capture.out -iw 720 -ih 576 -ow 640 -oh 480 -m 3 -bg
+Test 11. mxc_v4l2_overlay.out -iw 720 -ih 576 -ow 640 -oh 480 -m 3 -bg
 
-Test 12. mxc_v4l2_capture.out -iw 1280 -ih 720 -ow 640 -oh 480 -m 4 -bg
+Test 12. mxc_v4l2_overlay.out -iw 1280 -ih 720 -ow 640 -oh 480 -m 4 -bg
 
-Test 13. mxc_v4l2_capture.out -iw 1920 -ih 1080 -ow 640 -oh 480 -m 5 -bg
+Test 13. mxc_v4l2_overlay.out -iw 1920 -ih 1080 -ow 640 -oh 480 -m 5 -bg
 
-Test 14. mxc_v4l2_capture.out -iw 2592 -ih 1944 -ow 640 -oh 480 -m 6 -bg
+Test 14. mxc_v4l2_overlay.out -iw 2592 -ih 1944 -ow 640 -oh 480 -m 6 -bg
 
 | Expected Result |
 The image from the camera is displayed on the screen. Colors and image should be correct.
-- 
1.8.0

